DQMOffline/Trigger/python/HiggsMonitoring_Client_cff.py

```diff
@@ -1,4 +1,244 @@
+        "effic_eleEtaPhi_1	 'efficiency vs lead electron #eta-#phi; electron #eta ; electron #phi' eleEtaPhi_1_numerator       eleEtaPhi_1_denominator",
+        "effic_elePt_2       'efficiency vs sub-leading electron pt; electron pt [GeV]; efficiency' elePt_2_numerator	elePt_2_denominator",
+        "effic_eleEta_2       'efficiency vs sub-leading electron eta; electron eta ; efficiency' eleEta_2_numerator       eleEta_2_denominator",
+        "effic_elePhi_2       'efficiency vs sub-leading electron phi; electron phi ; efficiency' elePhi_2_numerator       elePhi_2_denominator",
+        "effic_elePt_2_variableBinning       'efficiency vs sub-leading electron pt; electron pt [GeV]; efficiency' elePt_2_variableBinning_numerator	elePt_2_variableBinning_denominator",
+        "effic_eleEta_2_variableBinning       'efficiency vs sub-leading electron eta; electron eta ; efficiency' eleEta_2_variableBinning_numerator       eleEta_2_variableBinning_denominator",
+        "effic_elePtEta_2	'efficiency vs sub-leading electron pt-#eta; electron pt [GeV]; electron #eta' elePtEta_2_numerator       elePtEta_2_denominator",
+        "effic_eleEtaPhi_2	 'efficiency vs sub-leading electron #eta-#phi; electron #eta ; electron #phi' eleEtaPhi_2_numerator       eleEtaPhi_2_denominator",
+        "effic_elePt_3       'efficiency vs trailing electron pt; electron pt [GeV]; efficiency' elePt_3_numerator       elePt_3_denominator",
+        "effic_eleEta_3       'efficiency vs trailing electron eta; electron eta ; efficiency' eleEta_3_numerator       eleEta_3_denominator",
+        "effic_elePhi_3       'efficiency vs trailing electron phi; electron phi ; efficiency' elePhi_3_numerator       elePhi_3_denominator",
+        "effic_elePt_3_variableBinning       'efficiency vs trailing electron pt; electron pt [GeV]; efficiency' elePt_3_variableBinning_numerator       elePt_3_variableBinning_denominator",
+        "effic_eleEta_3_variableBinning       'efficiency vs trailing electron eta; electron eta ; efficiency' eleEta_3_variableBinning_numerator       eleEta_3_variableBinning_denominator",
+        "effic_elePtEta_3       'efficiency vs trailing electron pt-#eta; electron pt [GeV]; electron #eta' elePtEta_3_numerator       elePtEta_3_denominator",
+        "effic_eleEtaPhi_3       'efficiency vs trailing electron #eta-#phi; electron #eta ; electron #phi' eleEtaPhi_3_numerator       eleEtaPhi_3_denominator",
+    ),
+    efficiencyProfile = cms.untracked.vstring(
+    	"effic_LeadElectronPt_vs_LS 'Electron p_T efficiency vs LS; LS; Electron p_T efficiency' eleVsLS_numerator eleVsLS_denominator"
+    ),
+)
+
+################################Triple Muon##########################
+triplemu12mu10mu5 = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/TriLepton/HLT_TripleMu_12_10_5/"),
```

Comment:
I added an update in the jira https://its.cern.ch/jira/browse/CMSHLT-1311 mentioning this PR.